cmd/cniplugins/passt-binding/pkg/plugin/plugin.go

```diff
@@ -100,17 +100,6 @@ func (c *cmd) CmdAddResult(args *skel.CmdArgs) (types.Result, error) {
 
 		netname := netConf.Args.Cni.LogicNetworkName
 		log.Printf("setup for logical network %s completed successfully", netname)
-
-		podLink, lerr := c.netlinkAdapter.ReadLink(primaryPodInterfaceName)
```

Comment:
Commenting here because github dont allow commenting on upper lines, the `primaryPodInterfaceName` seem redundant now, can we remove it?